Test failures out-of-the-box

On Ubuntu 19.04 I get a test failure out-of-the-box while following the quick start in the README.

CMake output:
```
-- The C compiler identification is GNU 8.3.0
-- The CXX compiler identification is GNU 8.3.0
-- Check for working C compiler: /usr/bin/gcc
-- Check for working C compiler: /usr/bin/gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/g++
-- Check for working CXX compiler: /usr/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- GPU acceleration is OFF
-- OMP acceleration is ON
-- MPI distribution is OFF
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- Compiling tutorial_example.c to executable demo
-- Found PythonInterp: /usr/local/bin/python3.4 (found version "3.4.9") 
-- Testing is ON
-- Configuring done
-- Generating done
-- Build files have been written to: /home/fergus/GitRepos/QuEST/build
```

Test output:
```
fergus@fergus:~/GitRepos/QuEST/build$ make test
Running tests...
Test project /home/fergus/GitRepos/QuEST/build
    Start 1: build
1/2 Test #1: build ............................   Passed    1.13 sec
    Start 2: unit
2/2 Test #2: unit .............................***Failed  Error regular expression found in output. Regex=[Error]  0.03 sec

50% tests passed, 1 tests failed out of 2

Total Test time (real) =   1.16 sec

The following tests FAILED:
	  2 - unit (Failed)
Errors while running CTest
make: *** [Makefile:73: test] Error 8
```

I cannot immediately see how to get a more verbose output to pin down what exactly is failing.

--- a/utilities/QuESTTest/QuESTCore.py
+++ b/utilities/QuESTTest/QuESTCore.py
@@ -266,7 +266,7 @@ class TestResults:
 
     def _write_term(self, *out, **kwargs):
         """ Wrapper for print with enforced flushing and silencing """
-        if self._printToScreen and root: print(*out, **kwargs, flush=True)
+        if self._printToScreen and root: print(*out, flush=True, **kwargs)
 
     def open_log(self, logFile):
         """ Open a new logFile associated with testResults """

